src/WebServerManager.cpp

```
@@ -813,6 +819,21 @@ void WebServerManager::handleApiPostConfig() {
         uint8_t ch = doc["u1_channel"] | (uint8_t)0;
         update.u1_channel = (ch <= 3) ? ch : 0;  // clamp invalid client input
     }
+    {
+        // Sent as a string so empty (= board default) is distinguishable from GPIO 0.
+        // Board-specific pin validation happens at the NVS boundary in saveToNVS.
+        const char* ledPinStr = doc["led_pin"] | "";
```

Clang (14.0.6)

[warning] variable 'ledPinStr' is not initialized

(cppcoreguidelines-init-variables)